docs/map.jinja.rst

```diff
@@ -22,11 +22,11 @@ Current best practice is to let ``map.jinja`` handle parameters from all sources
 
 
 For formula users
------------------
+=================
```

Comment:
Actually, the original heading is correct here (i.e. `-` heading 2, rather than `=` heading 1, which is only used for the title at the top).  Same as we're doing in `README.rst`.

If we agree upon that, then the rest of the headings changed below need to be demoted 1 level.

Reply:
Resolved in 40277fc582a62d2255b478fb0b880eda1f46d77a.